mne/epochs.py

```diff
@@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         # pick channels
         if picks is None:
             picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
-                               stim=False, eog=False, ecg=False,
+                               stim=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
```

Comment:
would there be a way to factorize this?

maybe a

DATA_PICKS = dict(eeg=True, meg=True ...)

in picks.py

and then in the files

picks = pick_types(info, **DATA_PICKS)